doc/build-android.md

```diff
@@ -22,4 +22,215 @@ This is an example command for a default build with no disabled dependencies:
\ No newline at end of file
+After the depends are built configure with one of the resulting prefixes and run `make && make apk` in `src/qt`.
+
+# Bitcoin Core WSL Ubuntu Android QML GUI
+
+This guide will walk you through the steps to build Bitcoin Core QML APKs using WSL Ubuntu 22.04 on Windows 11.
+
+## Prerequisites
+
+1. Install WSL Ubuntu 22.04 on Windows 11. You can find a comprehensive guide [here](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support#1-overview).
+
+## Installation Steps
+
+### Step 1: Update WSL Ubuntu 22.04
+
+After installing Ubuntu, run the following commands to update it:
+
+```bash
+sudo apt update
+sudo apt upgrade
+```
+
+### Step 2: Install Bitcoin Core Dependencies
+
+Next, install the necessary Bitcoin Core dependencies with the following commands:
+
+```bash
+sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git
+```
+
+Also install the QML specific dependencies:
+
+```bash
+sudo apt install qtdeclarative5-dev qtquickcontrols2-5-dev
+```
+
+### Step 3: Install Android Studio
+
+Follow the instructions below to install Android Studio and Android NDK on your system:
+
+1. Install OpenJDK-11-JDK:
+
+    ```bash
+    sudo apt install openjdk-11-jdk
+    ```
+
+2. Verify the installation by checking the java version:
+
+    ```bash
+    java --version
+    ```
+
+3. Install Android Studio using Snap:
+
+    ```bash
+    sudo snap install android-studio --classic
+    ```
+
+4. Run Android Studio:
+
+    ```bash
+    android-studio
+    ```
+
+You can also follow the full installation guide for Android Studio and Android NDK [here](https://linuxhint.com/install-android-studio-ubuntu22-04/).
+
+### Step 4: Install Android NDK
+
+To install Android NDK:
+
+1. With a project open in Android Studio, click `Tools > SDK Manager`.
+2. Click the `SDK Tools` tab.
+3. Select the `NDK (Side by side)` and `CMake` checkboxes.
+4. Click `OK`.
+5. A dialog box will tell you how much space the NDK package consumes on disk.
+6. Click `OK`.
+7. When the installation is complete, click `Finish`.
+
+You can find the full guide [here](https://developer.android.com/studio/projects/install-ndk).
+
+### Step 5: Check Android Device Architecture
+
+Before you proceed, ensure you check your Android Device's hardware architecture. Use usbip and adb for this. Detailed guide can be found [here](https://www.xda-developers.com/wsl-connect-usb-devices-windows-11/).
+
+1. Connect your Android device to your PC and enable USB debugging in Developer Options.
+2. Once usbip is installed, list all USB devices connected to your PC by running the following command as Administrator in `cmd`:
+
+    ```bash
+    usbipd wsl list
+    ```
+
+3. Note down the `BUSID` of the device you want to connect to WSL. Then run the following command, replacing `<busid>` with the `BUSID` of your device:
```

Comment:
Devices have to explicitly be exported using `usbip bind` before `usbipd` makes them available to other hosts.
So, before you run the `attach`command you need to bind the device: `usbipd bind -b <busid>`.

Reply:
good catch... it seems on my device the setting was already `SHARED`, however I'll add the `bind` suggestion in case the android device doesn't attach properly